tests/e2e/testcases/blockchain/configure/suite.go

```diff
@@ -0,0 +1,470 @@
+				"invalid-flag": "doesNotMatter",
+			},
+		)
+		gomega.Expect(err).Should(gomega.HaveOccurred())
+		gomega.Expect(output).Should(gomega.ContainSubstring("unknown flag: --invalid-flag"))
+	})
+
+	ginkgo.It("invalid blockchain conf path", func() {
+		output, err := commands.BlockchainConfigure(
+			utils.BlockchainName,
+			utils.TestFlags{
+				"chain-config": "invalidPath",
+			},
+		)
+		gomega.Expect(err).Should(gomega.HaveOccurred())
+		gomega.Expect(output).Should(gomega.ContainSubstring("open invalidPath: no such file or directory"))
+	})
+
+	ginkgo.It("invalid per node blockchain conf path", func() {
+		output, err := commands.BlockchainConfigure(
+			utils.BlockchainName,
+			utils.TestFlags{
+				"per-node-chain-config": "invalidPath",
+			},
+		)
+		gomega.Expect(err).Should(gomega.HaveOccurred())
+		gomega.Expect(output).Should(gomega.ContainSubstring("open invalidPath: no such file or directory"))
+	})
+
+	ginkgo.It("invalid subnet conf path", func() {
+		output, err := commands.BlockchainConfigure(
+			utils.BlockchainName,
+			utils.TestFlags{
+				"subnet-config": "invalidPath",
+			},
+		)
+		gomega.Expect(err).Should(gomega.HaveOccurred())
+		gomega.Expect(output).Should(gomega.ContainSubstring("open invalidPath: no such file or directory"))
+	})
+
+	ginkgo.It("invalid node conf path", func() {
+		output, err := commands.BlockchainConfigure(
+			utils.BlockchainName,
+			utils.TestFlags{
+				"node-config": "invalidPath",
+			},
+		)
+		gomega.Expect(err).Should(gomega.HaveOccurred())
+		gomega.Expect(output).Should(gomega.ContainSubstring("open invalidPath: no such file or directory"))
+	})
+
+	ginkgo.It("check default configs", func() {
+		output, err := utils.TestCommand(
+			utils.BlockchainCmd,
+			"deploy",
+			[]string{utils.BlockchainName},
+			utils.GlobalFlags{},
+			utils.TestFlags{
+				"local":             true,
+				"skip-icm-deploy":   true,
+				"skip-update-check": true,
+			},
+		)
+		gomega.Expect(output).Should(gomega.ContainSubstring("L1 is successfully deployed on Local Network"))
+		gomega.Expect(err).Should(gomega.BeNil())
+		blockchainID, err := utils.ParseBlockchainIDFromOutput(output)
+		gomega.Expect(err).Should(gomega.BeNil())
+		nodesInfo, err := utils.GetLocalClusterNodesInfo()
+		gomega.Expect(err).Should(gomega.BeNil())
+		checkBlockchainConfig(nodesInfo, blockchainID, defaultRPCTxFeeCap, nil)
+		checkSubnetConfig(nodesInfo, "")
+		checkNodeConfig(nodesInfo, -1)
+	})
+
+	ginkgo.It("set blockchain config", func() {
+		// set blockchain config before deploy
+		chainConfig := getBlockchainConfig(newRPCTxFeeCap1)
+		chainConfigPath, err := utils.CreateTmpFile(constants.ChainConfigFileName, []byte(chainConfig))
+		gomega.Expect(err).Should(gomega.BeNil())
+		_, err = commands.BlockchainConfigure(
+			utils.BlockchainName,
+			utils.TestFlags{
+				"chain-config": chainConfigPath,
+			},
+		)
+		gomega.Expect(err).Should(gomega.BeNil())
+		// deploy l1
+		output, err := utils.TestCommand(
```

Comment:
We should definitely figure out this part with `TestCommand` soon, because in this test it's not clear what we are testing exactly

Reply:
we are testing that configurations specified by `blockchain configure` are set during `blockchain deploy` and `network start`. I think this is the best way to check that `blockchain configure` does a proper and coherent work.

Reply:
changing to use new function commands.DeployBlockchain. anyway would like to know your idea on how to make more clear what it is being tested @artemanosov